definitions/variable/industry/tag_demand_sectors.yaml

```
@@ -0,0 +1,27 @@
+- Demand Sector:
+    - Electricity:
+        description: electricity generation sector
+    - Electricity|Biomass:
+          description: electricity generation with biomass power plants
+    - Electricity|Coal:
+          description: electricity generation with coal power plants
+    - Electricity|Gas:
+          description: electricity generation with gas power plants
+    - Electricity|Hydro:
+          description: electricity generation with hydro power plants
+    - Electricity|Nuclear:
+          description: electricity generation with nuclear power plants
+    - Electricity|Solar:
+          description: electricity generation with solar PV
```

In **common-definitions**, "Electricity|Solar" includes both PV and CSP. And maybe add sub-categories to distinguish if this is really necessary...
```suggestion
          description: electricity generation with solar photovoltaics and concentrated solar power
```